test/utils/dirft2d.hpp

```diff
@@ -1,40 +1,55 @@
-#include "finufft/finufft_core.h"
-#include <iostream>
+#pragma once
 
-// This is basically a port of dirft2d.f from CMCL package, except with
```

Comment:
Again I'd like you to keep historical comments. This one has a useful comment about prefactor.